docs/platform/howto/restrict-access.md

```diff
@@ -1,18 +1,22 @@
 ---
 title: Restrict network access to services
+sidebar_label: Restrict access
 ---
 
 import ActionsIcon from "@site/static/images/icons/more.svg";
 
 By default, Aiven services are publicly accessible, but you can restrict access to your service to a single IP, an address block, or any combination of both.
```

Comment:
I am thinking we can add here that if access from another service is needed, a service integration should be used, unless public access is allowed which is the default.